### ISSUE TITLE
SM: err.stack.split is not a function

### ISSUE THREAD

**Issue by ktrzeciaknubisa** [jxcore/jxcore#734]
One of our test cases (*test/jxcore-npm/test-modules-express-basic.js*) started to fail for SM:

```js
var assert = require("assert");
var express = require('express');  // 4.9.5.
var app = express();

assert.strictEqual(!!app, true, "Cannot get express() instance.");
```

**Comment by ktrzeciaknubisa** [comment]
Forgot about the error:

```bash
TypeError: err.stack.split is not a function (node.js 68:3)
    at Error.captureStackTrace (node.js:68:4)
    at getStack (/Users/nubisa_krzs/Documents/GitHub/jxcore/test/jxcore-npm/node_modules/express/node_modules/depd/index.js:382:3)
    at depd (/Users/nubisa_krzs/Documents/GitHub/jxcore/test/jxcore-npm/node_modules/express/node_modules/depd/index.js:119:15)
    at /Users/nubisa_krzs/Documents/GitHub/jxcore/test/jxcore-npm/node_modules/express/lib/router/index.js:21:17
    at Module.prototype._compile (module.js:621:8)
    at Module._extensions[".js"] (module.js:651:1)
    at Module.prototype.load (module.js:442:1)
    at Module._load (module.js:407:1)
    at Module.prototype.require (module.js:4
```

**Comment by ktrzeciaknubisa** [comment]
I still have something here:

```bash
TypeError: err.stack[3] is undefined (node.js 135:8)
    at tmp (node.js:135:9)
    at New_Error (node.js:144:14)
    at newError (node.js:149:19)
    at Error.captureStackTrace (node.js:54:17)
    at callSiteToString (/home/azureuser/Github/jxcore/test/jxcore-npm/node_modules/express/node_modules/depd/lib/compat/index.js:28:3)
    at get (/home/azureuser/Github/jxcore/test/jxcore-npm/node_modules/express/node_modules/depd/lib/compat/index.js:45:15)
    at /home/azureuser/Github/jxcore/test/jxcore-npm/node_modules/express/node_modules/depd/index.js:11:24
    at Module.prototype._compile (module.js:621:10)
    at Module._extensions[".js"] (module.js:6
```

**Comment by ktrzeciaknubisa** [comment]
This happens when one of modules used by express calls:

```js
var obj = {}
var prep = Error.prepareStackTrace

function prepareObjectStackTrace(obj, stack) {
  return stack
}

Error.prepareStackTrace = prepareObjectStackTrace
Error.stackTraceLimit = 2

// capture the stack
Error.captureStackTrace(obj)
```

I noticed, that if I set `Error.stackTraceLimit` to `4`, the problem goes away.

**Comment by ktrzeciaknubisa** [comment]
Fixed with 0e3a4c3a37916ea99002520030c24bf21a12c57a
